configs/ttalps_sfwriter_config.py

```diff
@@ -9,7 +9,7 @@
 from HistogramNormalizer import NormalizationType
 from Sample import Sample, SampleType
 
-# years = ["2023postBPix",]
+# years = ["2022postEE",]
```

Comment:
Also here, maybe time to remove this line, and we just run on all years by default?